app/views/template.scala.html

```diff
@@ -220,7 +220,7 @@
 
                                     @if(conf.link != null && conf.link.length > 0) {
                                         <li>
-                                            <a href="@conf.link">Conference home</a>
+                                            <a href="@conf.link">&#x1F5D7; Conference home</a>
```

Comment:
The icon is not properly rendered on Chromium or Firefox. See if you can use a bootstrap glyphicon instead, we have used these in the page before and they have worked well so far.